extension/manifest.json

```diff
@@ -25,6 +25,21 @@
         "48": "images/GearHeart_Mixed_47x47.png",
         "128": "images/GearHeart_Mixed_127x127.png"
     },
+    "commands": {
+        "toggle-feature": {
+            "suggested_key": {
+                "default": "Ctrl+Shift+F",
+                "mac": "MacCtrl+Shift+F"
+            },
+            "description": "Send a 'toggle-feature' event to the extension"
```

Comment:
what does this do?

Reply:
After loading your extension, whenever you press "Ctrl+Shift+F" on windows or "MacCtrl+Shift+F" on mac it will send a event which you can listen in your background.js file. I just used it for testing that and the code that I included in the background.js file is this :

chrome.commands.onCommand.addListener(function(command){
	console.log('onCommand event recieved for message: ', command);
});

Reply:
So is this test code, or is part of the implementation not committed? It's better to not submit work in progress code as a PR. We can talk about code options through the issue with links to your branch.

Reply:
If you want to have a listener in your background.js file everytime you press the key above it can be included in the code. I have included it on my computer but I haven't committed the implementation of background.js file.

Reply:
Could you explain what the use case is, what problem it would solve, or what the benefit would be?

Reply:
The "toggle-feature" is used for stuffs like whenever the key is clicked, a new tab will be opened or something like that. We can listen for the event in a script file and open a new tab using chrome.tabs API. It's just like a handler for each of the commands defined in the manifest. 
I just tested this feature by doing console.log(). If you think this is not needed I can remove it.

Reply:
If you'd like to use this approach to try to handle modifying the preferences using keyboard shortcuts, you are welcome to keep investigating that. However, we won't commit a console log to master. My suggestion would be to explore a bit more into whether or not it's feasible to modify the preferences using the keyboard shortcut. If it is, please provide that in the implementation. If that isn't feasible, or you are not able to complete that portion of the work. Please let me know and also back out these changes.

Reply:
Okay I've understood now. I'll make sure I don't include it in the final code. I assume that the rest of the code except the above part will solve the issue to open the panel from the keyboard. Hence, I request you to please check the rest of the code and merge it.

Reply:
This needs to be cleaned up before the PR can be merged.